packages/app/src/elements/settings-security.ts

```diff
@@ -689,11 +701,11 @@ export class SettingsSecurity extends StateMixin(Routing(LitElement)) {
                     <pl-toggle-button
                         class="transparent"
                         id="securityReportWeakToggle"
-                        .active=${app.settings.securityReportWeak}
+                        .active=${app.account?.settings.securityReport.weakPasswords || false}
                         .label=${html`<div class="horizontal center-aligning spacing layout">
                             <pl-icon icon="weak"></pl-icon>
                             <div>${$l("Weak Passwords")}</div>
-                        </div>`}
+                        </div>` as TemplateResult}
```

Comment:
Again, why is this cast necessary (it shouldn't be). I can also debug this for you if you're stuck.

Reply:
Because it has a variable inside, it's reported as `TemplateResult<1 | 2>` instead of `TemplateResult`. The app doesn't break without this, but `tslint` complains.

Reply:
> Because it has a variable inside, it's reported as TemplateResult<1 | 2> instead of TemplateResult

Not sure what you mean by that. Is there any documentation around this?

Reply:
I think the problem _might_ be related to https://github.com/runem/lit-analyzer/issues/154

This is what I'm talking about:

![Screenshot from 2022-08-02 07-31-02](https://user-images.githubusercontent.com/1239616/182307323-920dd7c1-19b6-4945-b417-ccb4b040d59f.png)

Reply:
Okay, let's table this for now, then. Maybe it'll get fixed upstream.